### ISSUE TITLE
Error running python-linters mypy

### ISSUE THREAD

**Issue by yochem** [elarivie/atom-python-linters#2]
# ⓘ Summary
After installing and opening I got the following error:
```
[Linter] Error running python-linters mypy mypyusage: mypy [-h] [-v] [-V] [more options; see below]
            [-m MODULE] [-p PACKAGE] [-c PROGRAM_TEXT] [files ...]
mypy: error: unrecognized arguments: --show-absolute-path
```

# 👣 Reproduction steps
1. Download python-linters
2. Open a python file
3. [edit] Save file (ctrl + s)
3. Receive error (I guess?)

# 🦄 Expected behaviour:
The linter works as expected, but the error shouldn't appear.

# 🐞 Actual behaviour:
Receiving the error mentioned above

# 📖 Stack trace (and Screenshot):
![image](https://user-images.githubusercontent.com/23235841/72830025-2310b780-3c80-11ea-8ee2-87460da3014d.png)

## 📁 Relevant data/files:
Please let me know which files/data would be relevant

## 💬 Comment:
Versions:
- **MacOS**: 10.15.2 (19C57)
- **Atom** : 1.43.0
    - **Electron**: 4.2.7
    - **Chrome**: 69.0.3497.128
    - **Node**: 10.11.0
- **atom-python-linters**: 0.0.3

I don't think I have any more to comment on. Please let me know if you need further information.

**Comment by elarivie** [comment]
Hello @yochem,

Thanks for your issue report.

The issue that you have is because your locally installed mypy is not the latest version.

The error mention that it does not recognize the command line argument _--show-absolute-path_
which was introduced in mypy v0.750
See mypy changelog entry: [Add option to print absolute file paths (Maxim Koltsov, PR 7754)](https://mypy-lang.blogspot.com/2019/11/mypy-0.html) 

**Note:** The latest mypy version as of today is 0.761

You should keep your lint tool up to date using the following command:

```shell
python3 -m pip install -U mypy
```

**Note:** -U is for Update

**Note:** I'll improve python-linters to detect this kind of situation and show user friendly messages about the situation instead of throwing an exception.


Let me know if it solves your issue.

**Comment by elarivie** [comment]
Mypy has a dependency on the python module 'typed-ast'

So to make sure both are updated you should run:

**python3 -m pip install -U mypy typed-ast**

**Comment by elarivie** [comment]
The fix is now available in python-linters v0.0.4

**Comment by yochem** [comment]
Thank you! The command `python3 -m pip install -U mypy typed-ast` indeed fixed the issue.
